api/vote.go

```diff
@@ -176,6 +176,10 @@ func (a *API) newVote(w http.ResponseWriter, r *http.Request) {
 		ErrResourceNotFound.Withf("could not get process: %v", err).Write(w)
 		return
 	}
+	// check that the process is ready to accept votes
+	if process.Status != types.ProcessStatusReady {
```

Comment:
Umm, we do have another way to check that (because READY does not mean the Sequencer accepts the votes for that process), it is implemented here https://github.com/vocdoni/davinci-node/blob/main/sequencer/ballot.go#L90

However, it would imply to bring the Sequencer to the API, which is not nice. So I'd just ignore it, make the Soft check you'r already doing, and let the Sequencer Ballot queue discard the vote.

Nonetheless, it would be nice if you can let a comment indicating this decision.